### PR TITLE
[MesheryPlayComponent] - Migrated class based play component to functional component.

### LINKED ISSUE #12169: [UI] Migrate Meshery Play Component to Functional Component with MUI v5 Styling and Sistent Component use
### Current Behavior
This [MesheryPlayComponent](https://github.com/meshery/meshery/blob/764d510d136d0e6640d7d5dc2f2e55c759c0be7c/ui/components/MesheryPlayComponent.js#L86) Component is a class-based component and it uses MUI v4's styling. These need to be updated for improved performance and consistency.

### Desired Behavior
- [ ]  migrate class-based component to a functional component
- [ ]  use [Sistent](https://github.com/layer5io/sistent) components for  MUI v4 to MUI v5 migration

Sistent is our design system that uses MUI v5, you can also refer to this [doc](https://docs.meshery.io/project/contributing/contributing-ui-sistent) to understand how to use Sistent.

### Acceptance Tests
The component must be updated without introducing any breaking changes in functionality or theming across all instances where it is used.

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Pratik-050**
**Notes for Reviewers**

This PR fixes #12169

Migrated from class based component to functional component without breaking the normal functionality of the component.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
@Pratik-050 
Thank you for your contributions! Let's discuss this during the Meshery Development call today at 9:00 AM CT. Adding it as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit?pli=1#) || [meeting link](https://meet.layer5.io/meshery).

File: ui/components/MesheryPlayComponent.js [hunk 1]
```diff
@@ -83,211 +83,184 @@ const styles = (theme) => ({
...
     return (
       <NoSsr>
-        <React.Fragment>
-          <div className={classes.playRoot}>
-            <Grid container>
-              <Grid item xs={12} className={classes.paneSection}>
-                <TextField
-                  select
-                  id="adapter_id"
-                  name="adapter_name"
```
Comment (Yashsharma1911) [inline ui/components/MesheryPlayComponent.js:245]: Why we removed or made changes here?

Comment (Pratik-050) [reply ui/components/MesheryPlayComponent.js:245]: Hello @Yashsharma1911 We didn't changed anything here I guess the part is not showing here properly. You can check from line 210-226.
I just removed the <React.Fragment>

File: ui/components/MesheryPlayComponent.js [hunk 2]
```diff
@@ -304,5 +277,5 @@ const mapStateToProps = (state) => {
 };
 
 export default withStyles(styles)(
```
Comment (Yashsharma1911) [inline ui/components/MesheryPlayComponent.js:279]: We can also remote withStyle and use styled component // @sudhanshutech

**Review by Yashsharma1911 — COMMENTED** [review]
Other then this it LGTM

File: ui/components/MesheryPlayComponent.js (hunk 2 above)
Comment (Pratik-050) [reply ui/components/MesheryPlayComponent.js:279]: Yeah that is being done by @Deathnote45 who is also assigned with me.
She's doing The MUI migration in a separate PR.

Comment (Yashsharma1911) [reply ui/components/MesheryPlayComponent.js:279]: Ohh I see, ok ok

**Review by Yashsharma1911 — APPROVED** [review]
With @sudhanshutech or @SAHU-01 approval we will be good to merge this PR

**Comment by Pratik-050** [comment]
Hi @sudhanshutech please have a look at this. Thanks!

File: ui/components/MesheryPlayComponent.js (hunk 2 above)
Comment (sudhanshutech) [reply ui/components/MesheryPlayComponent.js:279]: is @Deathnote45 is don for that other pr?

**Review by sudhanshutech — APPROVED** [review]

**Pull request merged**